pkgs/by-name/dr/druid/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, extensions ? { }
+, libJars ? [ ]
+, nixosTests
+, mysqlSupport ? true
+}:
+let
+  inherit (lib) concatStringsSep mapAttrsToList optionalString forEach;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache-druid";
+  version = "25.0.0";
+  src = fetchurl {
+    url = "https://dlcdn.apache.org/druid/${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-bin.tar.gz";
+    sha256 = "sha256-oG7vBq+hXNVkVWjr/bu2/e9x3X4PV9XDoDROT3gJMU0=";
+  };
+  mysqlConnector =
```

Comment:
This doesn't need the if-else. It won't be evaluated unless it is used in `installPhase`